dotnetv3/cross-service/README.md

```diff
@@ -1,9 +1,32 @@
 # AWS SDK for .NET 3.5+ cross-service examples
```

Comment:
(v3) is the version called out in line 4, do we need that instead of 3.5+ in the title?